src/c_code_generator.js

```diff
@@ -2892,7 +2931,7 @@ module.exports = async function generate(rom, functionName, fileName, bFastMode,
             {
```

Comment:
review line 2927, code += " fr.toS32(o, op0);\n", because 0xFFFFFFFF could be a valid negative or positive, on a field a "negative" number was considered when > p/2. Use example:
`0xFFFFFFFF => A
0 - A :JMPN // expected that only jump if A non zero`

Other example to check if A is zero, used:
`0xFFFFFFFF => A
A - 1 :JMPN // expected that only jump if A zero`

For it, 0xFFFFFFFF was a valid as negative as positive.